### PR TITLE
chansrv: fix warnings in pulse module

### PR DESCRIPTION
**Pull request by metalefty**
It was undiscovered due to pulse module is currently not tested by travis.

### CODE REVIEW

**Comment by proski** [comment]
We should incorporate audio into the build.

The `pa_usec_t` type is defined as `uint64_t`. We cannot assume that it's `long`. The standard approach is to cast the argument to `unsigned long long` and use `%llu` format.

**Comment by metalefty** [comment]
I already have a Makefile to build pulse module. I'll brush it up and integrate into travis build.

**Comment by proski** [comment]
It's not just Travis. It should be built and installed as part of the build. Then Travis would pick it up automatically.

**Comment by metalefty** [comment]
Sure. But pulseaudio's configured source is required to build. The build process will be different depending on distro.

**Comment by jsorg71** [comment]
I've tried to push the pulseaudio guys to release a development package for building sinks and sources out of tree.  Sometime like pulseaudio-dev for Debian.
They did not seem very interested.
It's too bad, they have a really nice module system.

**Comment by metalefty** [comment]
Anyway, let's fix warnings first. If the change looks good, please merge.

**Comment by jsorg71** [comment]
+1
Looks good to me, remember, in this file we are trying to match pulseaudio's coding style.

File: sesman/chansrv/pulse/module-xrdp-sink.c [hunk 1]
```diff
@@ -171,12 +171,12 @@ static void sink_update_requested_latency_cb(pa_sink *s) {
 
     u->block_usec = BLOCK_USEC;
     //u->block_usec = pa_sink_get_requested_latency_within_thread(s);
-    pa_log("1 block_usec %d", u->block_usec);
+    pa_log("1 block_usec %llu", (unsigned long long)u->block_usec);
```
Comment (proski) [inline sesman/chansrv/pulse/module-xrdp-sink.c]: Please use space after cast here and elsewhere. I checked pulseaudio 10.0 sources, they use space too.

Comment (metalefty) [inline sesman/chansrv/pulse/module-xrdp-sink.c]: ok. I always forget put a space after cast :disappointed:

Comment (metalefty) [inline sesman/chansrv/pulse/module-xrdp-sink.c]: done.

**Pull request merged**